script/Arcueid/Day 9/09_00_ARC09_1.txt

```diff
@@ -121,32 +121,33 @@ Akiha is crying.
 [sha:6a24cf39142588eb5959b56d5e2c5860e3e15eb7]{
 -- Page 4, Offset 21316. Mods: @k, @e.
 -- おとなしくて、いつもボクの後についてきた秋葉が、
-Akiha, who had always been quiet and followed us around,
+Akiha, who had always meekly followed us around,
 }
 [sha:b971cd482cecd507e899764af85b5b36db40e755]{
 -- Page 4, Offset 21317.
 -- ぽろぽろと泣いている。
-is sobbing on the floor.
+is sobbing uncontrollably.
 }
 [sha:ae14460d339d1db596170eb67a6eb9ef74616931]{
 -- Page 5, Offset 21318.
 -- 広場には子供がひとり倒れている。
-There is a single child collapsed on the square.
+A child lies crumpled in the clearing.
 }
 [sha:9b13c7cf5b90dbcd5fdd4990fe9f41bdac9d2939]{
 -- Page 5, Offset 21319.
 -- 白いシャツが真っ赤に染まって、ぴくりとも動かない。
-Wearing a white shirt drenched in deep red, not even moving a single twitch.
+A deep red stain has blossomed on its white shirt.
+ It doesn't move so much as an eyebrow.
 }
 [sha:c8d80ad0594814cdb06ede76f65f678eaaf5ca02]{
 -- Page 5, Offset 21320.
 -- ボクは、それを見下ろしている。
-I look down upon it.
+I'm gazing down at the scene before me.
 }
 [sha:5a3b9ab142cbd637aef5fa4d5995d6b55c2a071c]{
 -- Page 5, Offset 21321.
 -- 両手は倒れている子供と同じように赤い。
-My hands are red just like the child on the floor.
+My hands are red, just like the child now collapsed in a heap on the floor.
```

Comment:
just like the child, collapsed in a heap on the floor

Reply:
Hmm, not sure I agree on that one. It makes it feel like the middle can be cut, which would result in 
> My hands are red, just like the child, collapsed in a heap on the floor.
> My hands are red, collapsed in a heap on the floor.

which confuses the subject.
I also would prefer that this line and the line that's basically the same but s/子供/イキモノ match syntax, and in that line I think the effect is even more awkward:
> My hands are red, with the blood of the living being, collapsed in a heap on the floor.

Reply:
the double comma isn't a clarification or description though, it's just two commas. I don't feel strongly either way but I also don't really think that interpretation is right.

Reply:
As in the JP interpretation?

Reply:
No, the interpretation of my edit. I think the interpretation of the jp is correct

Reply:
I definitely see what it's going for, it's just a more vexing parse. I'd be cool with
> My hands are red. Just like the child, collapsed ...

since it is a lot harder to mis-read?

Reply:
sure, that's fine by me